src/app/helptext/network/interfaces/interfaces-form.ts

```diff
@@ -24,7 +24,7 @@ export default {
   ],
 
   int_interface_placeholder: T('NIC'),
-  int_interface_tooltip: T('Enter the FreeBSD device name of the\
+  int_interface_tooltip: T('Enter the device name of the\
  interface. This cannot be changed after the interface is created.'),
   int_interface_validation: [Validators.required],
```

Comment:
What about place where links from www.freebsd.org are used? Are those alright?

Reply:
Addressed in another PR.